### ISSUE TITLE
Is there an option NOT to insert spaces with the TAB key?

### ISSUE THREAD

**Issue by manngo** [Zettlr/Zettlr#2624]
### Checklist

- [X] Zettlr does not possess this feature (if it does, but the feature doesn't work out for you, propose an enhancement instead.)
- [X] Zettlr's current features cannot be used to approximate this behaviour (otherwise, propose an enhancement instead.)
- [X] I have searched the documentation to make sure I did not overlook it.
- [X] I have searched both open and closed issues to see if someone has already proposed this.

### Feature Proposal

I don’t know whether this option exists, bu I couldn’t find any information.The issue at [Editor inserts tabs instead of spaces · Issue #1097 · Zettlr/Zettlr](https://github.com/Zettlr/Zettlr/issues/1097) is the closest I could find.

I understand that some developers like spaces. Is there an option to keep genuine tabs in the document?

### Example Workflow

I write a lot of coding-related documents. Without going into any holy wars, I prefer to indent with tabs instead of spaces.

### Caveats

I can’t see any issues.

### Do you wish to attempt implementing this yourself?

No

### Platforms

- [X] Windows
- [X] macOS
- [X] Linux

**Comment by nathanlesage** [comment]
> Without going into any holy wars,

\*takes out the two-handed sword\*

jk, tab vs. spaces behavior is awful at times, because at one point you need genuine tabs, at the other just spaces. Even without taking into account the spacetab wars.

**Comment by arfathyahiya** [comment]
Stumbled across this issue after beating the shit out of my head. 
So I had SQL query which I created in docmost editor using codeblock. 

I had used tab and funnily enough my sql editor did not highlight anything and shit kept throwing error saying function did not exists
```
    AVG(column_name)
```
only after pasting the same query in a different editor I found out it was this character `U+00a0` causing the issue. 

Is there any way to make docmost use spaces instead of tabs in codeblocks?

**Comment by nathanlesage** [comment]
![Image](https://github.com/user-attachments/assets/7d97fde6-8134-4ca5-aaff-dd540f736280)

This setting is part of the editor settings, and should do the trick. I'll close this issue as it has been fixed. However, if there is buggy behavior with said setting, and it doesn't properly switch modes between spaces and tabs, please open a new issue indicating this. Thanks!
